airbyte-integrations/connectors/source-bing-ads/source_bing_ads/spec.json

```diff
@@ -0,0 +1,90 @@
+{
+  "documentationUrl": "https://docs.airbyte.io/integrations/sources/bing-ads",
+  "connectionSpecification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Bing Ads Spec",
+    "type": "object",
+    "required": [
+      "accounts",
```

Comment:
This logic could be replaced with the simpler one, where we check whether the list is `empty` or having the list of `account_ids`. Check this modified `spec.json`:

```
{
  "documentationUrl": "https://docs.airbyte.io/integrations/sources/bing-ads",
  "connectionSpecification": {
    "$schema": "http://json-schema.org/draft-07/schema#",
    "title": "Bing Ads Spec",
    "type": "object",
    "required": [
      "account_ids",
      "client_id",
      "client_secret",
      "customer_id",
      "developer_token",
      "refresh_token",
      "user_id"
    ],
    "additionalProperties": false,
    "properties": {
      "account_ids": {
        "type": "array",
        "description": "Leave default to fetch the data from all the accounts associated with the current User, otherwise - provide the list of account_ids.",
        "items": {"type": "string"},
        "default": []
      },
      "client_id": {
        "type": "string",
        "description": "ID of your Microsoft Advertising client application.",
        "airbyte_secret": true
      },
      "client_secret": {
        "type": "string",
        "description": "Secret of your Microsoft Advertising client application.",
        "airbyte_secret": true
      },
      "customer_id": {
        "type": "string",
        "description": "User's customer ID."
      },
      "developer_token": {
        "type": "string",
        "description": "Developer token associated with user.",
        "airbyte_secret": true
      },
      "refresh_token": {
        "type": "string",
        "description": "The long-lived Refresh token received via grant_type=refresh_token request.",
        "airbyte_secret": true
      },
      "user_id": {
        "type": "string",
        "description": "Unique user identifier."
      }
    }
  }
}
```

Reply:
we decided to use dropdowns here to have better UX

Reply:
ok